src/helpers/pivot/pivot_menu_items.ts

```diff
@@ -15,3 +15,33 @@ export const pivotProperties: ActionSpec = {
   },
   icon: "o-spreadsheet-Icon.PIVOT",
 };
+
+export const FIX_FORMULAS: ActionSpec = {
+  name: _t("Convert to Individual Formulas"),
+  execute(env) {
+    const position = env.model.getters.getActivePosition();
+    const cell = env.model.getters.getCorrespondingFormulaCell(position);
+    const pivotId = env.model.getters.getPivotIdFromPosition(position);
+    if (!cell || !pivotId) {
+      return;
+    }
+    const { sheetId, col, row } = env.model.getters.getCellPosition(cell.id);
+    const pivot = env.model.getters.getPivot(pivotId);
+    pivot.init();
```

Comment:
The menu should be hidden if the pivot is not valid, I added it. But this code is still needed to ensure that the pivot is loaded, as it could be invalided between the menu is opened and the action is selected